engine/flink/management/periodic/src/main/scala/pl/touk/nussknacker/engine/management/periodic/db/PeriodicProcessesRepository.scala

```diff
@@ -117,7 +118,7 @@ class SlickPeriodicProcessesRepository(db: JdbcBackend.DatabaseDef,
       id = PeriodicProcessId(-1),
       processName = deploymentWithJarData.processVersion.processName,
       processVersionId = deploymentWithJarData.processVersion.versionId,
-      processJson = deploymentWithJarData.graphProcess.marshalled,
+      processJson = ProcessMarshaller.toJson(deploymentWithJarData.canonicalProcess).noSpaces,
```

Comment:
Sometimes we use noSpaces other times we use space2 .. This can generate errors at equaling.. Can we pass here Canonical Process or use some method which return normalized json string? Maybe this example is't exactly what I've described, but still I think we should use one way to present json string.

Reply:
As we talked, it is because of performance reasons - I'll try to move this `CanonicalProcess` to entities and in next change add easier way to transition between canonicalprocess and its json representation